Files.Launcher/Program.cs

```diff
@@ -499,26 +499,24 @@ await Win32API.SendMessageAsync(connection, new ValueSet()
                         var flc = new List<ShellFileItem>();
                         try
                         {
-                            using (var shellFolder = new ShellFolder(folderPath))
-                            {
-                                foreach (var folderItem in shellFolder)
-                                {
-                                    try
-                                    {
-                                        var shellFileItem = GetShellFileItem(folderItem);
-                                        flc.Add(shellFileItem);
-                                    }
-                                    catch (FileNotFoundException)
-                                    {
-                                        // Happens if files are being deleted
-                                    }
-                                    finally
-                                    {
-                                        folderItem.Dispose();
-                                    }
-                                }
-                            }
-                        }
+							using var shellFolder = new ShellFolder(folderPath);
```

Comment:
There's a lot of unnecessarily large indents throughout the changes, like right here. Could you please fix them?